exasol/nb_connector/cli/processing/option_set.py

```diff
@@ -128,7 +127,7 @@ def get_scs(scs_file: Path) -> Secrets:
     return Secrets(scs_file, scs_password)
 
 
-def get_option_set(scs_file: Path) -> OptionSet | None:
+def get_option_set(scs_file: Path) -> OptionSet:
     """
     Return an instance of an OptionSet if the SCS contains a proper
     backend selection. Otherwise report an error and return None.
```

Comment:
Please update the docstring according to your latest changes.

Reply:
See nest push.